wiki/Mapping_and_Modding_Timeline/en.md

```diff
@@ -482,6 +486,47 @@ Mapping and modding systems are constantly improving. The **Mapping and modding
+- **2021-03-15:** Requirements for re-joining the Beatmap Nominators were loosened.
+  - All applicants needed to undergo the application, but returning group members had shorter cooldown options and were allowed to provide 1-2 months of modding activity instead of the previous 3 month standard. <!-- https://github.com/ppy/osu-wiki/pull/5027/files -->
+- **2021-03-16:** Beatmap Nominators were given automatically renewing osu!supporter.
+  - This change was made in response to a lack of incentives for dedicated members of the mapping scene. <!-- https://osu.ppy.sh/community/forums/topics/1268711?n=58 -->
+- **2021-03-16:** Ranked maps per day per mode limit increased from 8 to 10 to compensate for an overflowing Qualified beatmaps category.
+
+### April
+
+- **2021-04-08:** Ranking Criteria Rule disallowing multiple Qualified beatmaps of the same song was removed.
+  - Concerns about malicious disqualification practices were relieved by this change.
+
+### May
+
+- **2021-05-12:** The Nomination Assessment Team began a trial for new Beatmap Nominator evaluations. <!-- https://osu.ppy.sh/community/forums/topics/1400803?n=1 -->
+  - This trial only included osu! members and concluded in August.
+  - A subset of Beatmap Nominators were given evaluation roles equal to the NAT, allowing them to contribute to and decide the outcomes of BN applications and current BN evaluations.
+  - The first batch of trial members included ~10 users, while the second batch included ~20 users.
+- **2021-05-13:** Option to change difficulty owner was added to the Beatmap Discussion page. <!-- https://github.com/ppy/osu-web/pull/7517 -->
+  - This allowed guest difficulty creators to resolve issues on their beatmaps. Only the mapset host could resolve issues prior to this change, which was a major frustration to mappers.
+
+### June
+
+- **2021-06-14:** The Global Moderation Team began accepting applications from mapping-focused users to bridge the growing tension between the GMT and mapping/modding scenes. <!-- https://discord.com/channels/316154420591067136/316586967171203075/854061295702769694 / document any effects of this later -->
+
+### August 
+
+- **2021-08-14:** osu!taiko began a trial of Beatmap Nominator evaluations mimicking osu!'s trial from 2021-05-12.
+
+### September
+
+- **2021-09-11:** peppy changed Ranking Criteria rule about the Source field. <!-- https://github.com/ppy/osu-wiki/commit/02dee3fff09c3e890dc408271bb3d2d93c3c5b2d -->
+  - Mapping community members were unhappy with the lack of public discussion behind this change and later made adaptations to close new loopholes. <!-- https://osu.ppy.sh/community/forums/topics/1417764?n=1 -->
+- **2021-09-18:** osu!'s first community meeting began, one of its primary topics being [concerns with osu!mania's Ranking Criteria](https://osu.ppy.sh/community/forums/topics/1388182?n=1).
```

Comment:
iirc peppy was saying on Twitter that this is not the very first ever Community Meeting tho, as it was already a thing in the past apparently :

http://twitter.com/ppy/status/1437625042561474560

Reply:
updated wording to cover this